test/libyul/evmCodeTransform/eof_swap255.yul

```diff
@@ -0,0 +1,2827 @@
+//     /* "":8719:8736   */
+//   swapn{211}
+//     /* "":8732:8735   */
+//   0xea
+//     /* "":8719:8736   */
+//   calldataload
+//     /* "":8757:8774   */
+//   swapn{213}
+//     /* "":8770:8773   */
+//   0xeb
+//     /* "":8757:8774   */
+//   calldataload
+//     /* "":8795:8812   */
+//   swapn{215}
+//     /* "":8808:8811   */
+//   0xec
+//     /* "":8795:8812   */
+//   calldataload
+//     /* "":8833:8850   */
+//   swapn{217}
+//     /* "":8846:8849   */
+//   0xed
+//     /* "":8833:8850   */
+//   calldataload
+//     /* "":8871:8888   */
+//   swapn{219}
+//     /* "":8884:8887   */
+//   0xee
+//     /* "":8871:8888   */
+//   calldataload
+//     /* "":8909:8926   */
+//   swapn{221}
+//     /* "":8922:8925   */
+//   0xef
+//     /* "":8909:8926   */
+//   calldataload
+//     /* "":8947:8964   */
+//   swapn{223}
+//     /* "":8960:8963   */
+//   0xf0
+//     /* "":8947:8964   */
+//   calldataload
+//     /* "":8985:9002   */
+//   swapn{225}
+//     /* "":8998:9001   */
+//   0xf1
+//     /* "":8985:9002   */
+//   calldataload
+//     /* "":9023:9040   */
+//   swapn{227}
+//     /* "":9036:9039   */
+//   0xf2
+//     /* "":9023:9040   */
+//   calldataload
+//     /* "":9061:9078   */
+//   swapn{229}
+//     /* "":9074:9077   */
+//   0xf3
+//     /* "":9061:9078   */
+//   calldataload
+//     /* "":9099:9116   */
+//   swapn{231}
+//     /* "":9112:9115   */
+//   0xf4
+//     /* "":9099:9116   */
+//   calldataload
+//     /* "":9137:9154   */
+//   swapn{233}
+//     /* "":9150:9153   */
+//   0xf5
+//     /* "":9137:9154   */
+//   calldataload
+//     /* "":9175:9192   */
+//   swapn{235}
+//     /* "":9188:9191   */
+//   0xf6
+//     /* "":9175:9192   */
+//   calldataload
+//     /* "":9213:9230   */
+//   swapn{237}
+//     /* "":9226:9229   */
+//   0xf7
+//     /* "":9213:9230   */
+//   calldataload
+//     /* "":9251:9268   */
+//   swapn{239}
+//     /* "":9264:9267   */
+//   0xf8
+//     /* "":9251:9268   */
+//   calldataload
+//     /* "":9289:9306   */
+//   swapn{241}
+//     /* "":9302:9305   */
+//   0xf9
+//     /* "":9289:9306   */
+//   calldataload
+//     /* "":9327:9344   */
+//   swapn{243}
+//     /* "":9340:9343   */
+//   0xfa
+//     /* "":9327:9344   */
+//   calldataload
+//     /* "":9365:9382   */
+//   swapn{245}
+//     /* "":9378:9381   */
+//   0xfb
+//     /* "":9365:9382   */
+//   calldataload
+//     /* "":9403:9420   */
+//   swapn{247}
+//     /* "":9416:9419   */
+//   0xfc
+//     /* "":9403:9420   */
+//   calldataload
+//     /* "":9441:9458   */
+//   swapn{249}
+//     /* "":9454:9457   */
+//   0xfd
+//     /* "":9441:9458   */
+//   calldataload
+//     /* "":9479:9496   */
+//   swapn{251}
+//     /* "":9492:9495   */
+//   0xfe
+//     /* "":9479:9496   */
+//   calldataload
+//     /* "":9517:9534   */
+//   swapn{253}
+//     /* "":9530:9533   */
+//   0xff
+//     /* "":9517:9534   */
+//   calldataload
+//     /* "":9543:9556   */
+//   swapn{255}
```

Comment:
Not sure how valuable this test case really is... but here we have ``swapn{255}``... generating one more variable in the test case, correctly triggers a swap-too-deep.